sources/packages/backend/apps/api/src/route-controllers/application/_tests_/application.students.controller.getCompletedApplicationDetails.e2e-spec.ts

```diff
@@ -540,6 +560,338 @@ describe("ApplicationStudentsController(e2e)-getCompletedApplicationDetails", ()
+        {
+          applicationStatus: ApplicationStatus.Completed,
+          offeringIntensity: OfferingIntensity.partTime,
+          firstDisbursementInitialValues: {
+            coeStatus: COEStatus.completed,
+            disbursementScheduleStatus: DisbursementScheduleStatus.Pending,
+          },
+        },
+      );
+      const [firstDisbursement] =
+        application.currentAssessment.disbursementSchedules;
+
+      application.student.sinValidation.isValidSIN = false;
+      await db.student.save(application.student);
+
+      const endpoint = `/students/application/${application.id}/completed`;
+      const token = await getStudentToken(
+        FakeStudentUsersTypes.FakeStudentUserType1,
+      );
+
+      // Act/Assert
+      await request(app.getHttpServer())
+        .get(endpoint)
+        .auth(token, BEARER_AUTH_TYPE)
+        .expect(HttpStatus.OK)
+        .expect({
+          firstDisbursement: {
+            coeStatus: firstDisbursement.coeStatus,
+            disbursementScheduleStatus:
+              firstDisbursement.disbursementScheduleStatus,
+          },
+          assessmentTriggerType: application.currentAssessment.triggerType,
+          hasBlockFundingFeedbackError: false,
+          eCertFailedValidations: [ECertFailedValidation.InvalidSIN],
+        });
+    },
+  );
+
+  it(
+    "Should get application details with ecert failed validations array having lifetime maximum CSLP when " +
+      "the life time maximums CSLP value is less than the sum of disbursement CSLP value amount " +
+      "and the latest CSLP balance and the offering intensity is part time.",
+    async () => {
+      // Arrange
+      const student = await saveFakeStudent(db.dataSource);
+      const msfaaNumber = createFakeMSFAANumber(
+        {
+          student,
+        },
+        {
+          msfaaState: MSFAAStates.Signed,
+        },
+      );
+      await db.msfaaNumber.save(msfaaNumber);
+
+      // Mock user services to return the saved student.
+      await mockUserLoginInfo(appModule, student);
+
+      const application = await saveFakeApplicationDisbursements(
+        appDataSource,
+        { student, msfaaNumber },
+        {
+          applicationStatus: ApplicationStatus.Completed,
+          offeringIntensity: OfferingIntensity.partTime,
+          firstDisbursementInitialValues: {
+            coeStatus: COEStatus.completed,
+            disbursementScheduleStatus: DisbursementScheduleStatus.Pending,
+          },
+        },
+      );
+      const [firstDisbursement] =
+        application.currentAssessment.disbursementSchedules;
+
+      const disbursementCSLP = firstDisbursement.disbursementValues.find(
+        (item) => item.valueCode === CANADA_STUDENT_LOAN_PART_TIME_AWARD_CODE,
+      );
+      disbursementCSLP.valueAmount =
+        application.currentAssessment.workflowData.dmnValues
+          .lifetimeMaximumCSLP + 1;
+      await db.disbursementValue.save(disbursementCSLP);
```

Comment:
Not a blocker but it can I would suggest using the `createFakeDisbursementValue` as shown below.
```ts
const application = await saveFakeApplicationDisbursements(
        appDataSource,
        {
          student,
          msfaaNumber,
          disbursementValues: [
            createFakeDisbursementValue(
              DisbursementValueType.CanadaLoan,
              CANADA_STUDENT_LOAN_PART_TIME_AWARD_CODE,
              10001,
            ),
          ],
        },
        {
          applicationStatus: ApplicationStatus.Completed,
          offeringIntensity: OfferingIntensity.partTime,
          firstDisbursementInitialValues: {
            coeStatus: COEStatus.completed,
            disbursementScheduleStatus: DisbursementScheduleStatus.Pending,
          },
        },
      );
```